1.0/profile.ttl

```diff
@@ -0,0 +1,64 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX prof: <http://www.w3.org/ns/dx/prof/>
+PREFIX role: <http://www.w3.org/ns/dx/prof/role/>
+PREFIX sdo: <https://schema.org/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+
+<http://www.opengis.net/def/geosparql>
+    a prof:Profile ;
+    owl:versionIRI <http://www.opengis.net/def/geosparql/1.0> ;   
+    dcterms:created "2020-11-20"^^xsd:date ;
+    dcterms:modified "2020-12-19"^^xsd:date ;
+    dcterms:issued "2021"^^xsd:gYear ;
+    dcterms:publisher <https://ror.org/00fsdxs93> ;
+    dcterms:title "GeoSPARQL 1.1 Profile" ;
```

Comment:
should be:  `dcterms:title "GeoSPARQL 1.0 Profile" ;`

Reply:
Fixed with Commit https://github.com/opengeospatial/ogc-geosparql/pull/58/commits/a571175864dae1477076ca0c197bae2688ced0e1